cpp/IceBox/greeter/README.md

```diff
@@ -5,29 +5,22 @@ This demo shows how to create an IceBox service in C++.
 You can build the client application and IceBox service with:
 
 ```shell
-cmake -B build
-cmake --build build --config Release
+cmake -B build -S . -G Ninja -DCMAKE_BUILD_TYPE=Release
```

Comment:
I find this cmake command pretty extreme. We can't move some of these options to CMakeLists.txt?